app/controllers/sessions/session.ts

```diff
@@ -2,11 +2,20 @@ import Controller from '@ember/controller';
 import { ModelFrom } from '../../lib/type-utils';
 import SessionSessionRoute from '../../routes/sessions/session';
 import { sortObjectsByTitle } from 'frontend-burgernabije-besluitendatabank/utils/array-utils';
+import AgendaItem from 'frontend-burgernabije-besluitendatabank/models/agenda-item';
 
 export default class SessionsSessionController extends Controller {
   declare model: ModelFrom<SessionSessionRoute>;
 
   get agendaItemsSorted() {
-    return this.model.agendaItems.toArray().sort(sortObjectsByTitle);
+    let agendaItems = this.model.hasMany('agendaItems').value() as
```

Comment:
I switched to the hasMany reference API since it's not safe to access async relationships in getters in a sync way like the previous implementation. This version assumes the relationship is loaded already though.